conbench/tests/api/_expected_docs.py

```diff
@@ -1075,9 +1075,9 @@
             },
             "BenchmarkUpdate": {
                 "properties": {
-                    "is_step_change": {
-                        "description": "Is this result the first result of a sufficiently 'different' distribution than the result on the previous commit (for the same hardware/case/context)? That is, when evaluating whether future results are regressions or improvements, should we treat data from before this result as incomparable?",
-                        "type": "boolean",
+                    "change_annotations": {
+                        "description": "Post-analysis annotations about this BenchmarkResult that\ngive details about whether it represents a change, outlier, etc. in the overall\ndistribution of BenchmarkResults.\n\n\nThis endpoint will only update the user-specified keys, and leave the rest alone. To\ndelete an existing key, set the value to null.\n",
```

Comment:
We should add details about why one might set this field (and we should also be explicit that — for now — all details inside of it are entirely ignored). The prior boolean had "That is, when evaluating whether future results are regressions or improvements, should we treat data from before this result as incomparable?" which is IMO a decent description + vague enough for us to get to the end of this project without needing to come back and change it but still be meaningful to folks using conbench to have some idea of what to do with this.

Reply:
Agreed - I will do this in #575 when I write code that needs a specific element of the JSON.